aemo.gemspec

```diff
@@ -23,11 +23,11 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 1.9.3'
 
   # Production Dependencies
-  s.add_dependency 'json',      '~> 2.1'
+  s.add_dependency 'json',      '>= 1.7.5'
```

Comment:
Why >= 1.7.5 @jeznag ?

Reply:
It didn't work with energylink otherwise

Reply:
>< ok :)